ui/search/nuxeo-results-view.js

```diff
@@ -0,0 +1,453 @@
+                <paper-button noink on-tap="_clear" class$="[[_hideCounter]]">
+                  [[i18n('command.clear')]]
+                </paper-button>
+              </div>
+            </div>
+            <iron-collapse id="collapse" opened="{{opened}}">
+              <nuxeo-search-form-layout
+                id="form"
+                href-base="[[hrefBase]]"
+                provider="[[provider]]"
+                search-name="[[searchName]]"
+                aggregations="[[aggregations]]"
+                params="[[_params]]"
+                on-search-form-layout-changed="_formChanged"
+              ></nuxeo-search-form-layout>
+            </iron-collapse>
+            <div class="buttons" hidden$="[[!opened]]">
+              <paper-button noink class="clear secondary" on-tap="_clear" disabled$="[[loading]]">
+                [[i18n('command.clear')]]
+              </paper-button>
+              <div class="horizontal layout flex end-justified">
+                <paper-button noink class="primary search" on-tap="_search" hidden$="[[auto]]" disabled$="[[loading]]">
+                  [[i18n('command.search')]]
+                  <template is="dom-if" if="[[loading]]">
+                    <paper-spinner-lite active></paper-spinner-lite>
+                  </template>
+                </paper-button>
+              </div>
+            </div>
+          </div>
+        </template>
+
+        <template is="dom-if" if="[[visible]]">
+          <nuxeo-search-results-layout
+            id="results"
+            href-base="[[hrefBase]]"
+            search-name="[[searchName]]"
+            nx-provider="[[_nxProvider]]"
+            on-navigate="_navigateFromSearch"
+            on-results-changed="_resultsChanged"
+          ></nuxeo-search-results-layout>
+        </template>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-results-view';
+    }
+
+    static get properties() {
+      return {
+        /**
+         * The id of the `nuxeo-page-provider` instance used to perform the search.
+         */
+        provider: {
+          type: String,
+        },
+        /**
+         * The number of results per page.
+         */
+        pageSize: {
+          type: Number,
+          value: 40,
+        },
+        /**
+         * The query parameters passed on to `provider`.
+         */
+        params: {
+          type: Object,
+          observer: '_paramsChanged',
+          value: {},
+        },
+        /**
+         * List of content enrichers passed on to `provider`.
+         * Already set by default are thumbnail, permissions and highlight.
+         */
+        enrichers: {
+          type: String,
+          value: 'thumbnail, permissions, highlight',
+        },
+        /**
+         * The headers passed on to `provider`.
+         * Already set by default are 'fetch-document': 'properties' and 'translate-directoryEntry': 'label'.
```

Comment:
is the second part of this comment helpful since the default values are already in the property?
```suggestion

```

Reply:
I'm not sure if this wasn't added so that it would show up on Designer. Worth checking the blame.

Reply:
This info is shown in Designer.
![Screenshot 2021-04-14 at 16 10 49](https://user-images.githubusercontent.com/4223240/114734386-115f7800-9d3c-11eb-81e0-e74a589c2155.png)

Reply:
The default is still fully visible on the input in right, right?

Reply:
If it is just a placeholder then I think we really need to keep this as is.

Reply:
The JSDoc part goes into the context bubble and the actual default value of the property is used on the input placeholder. Since the input placeholder is a bit flawed (you can't see the whole thing), you actually need this duplicated info.